get-started/03_updating_app.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Updating our Application"
+keywords: get started, setup, orientation, quickstart, intro, concepts, containers, docker desktop
+description: Making changes to our example learning application
+---
+
+
+As a small feature request, we've been asked by the product team to
+change the "empty text" when we don't have any todo list items. They
+would like to transition it to the following:
+
+> You have no todo items yet! Add one above!
+
+Pretty simple, right? Let's make the change.
+
+## Updating our Source Code
+
+1. In the `src/static/js/app.js` file, update line 56 to use the new empty text.
+
+    ```diff
+    -                <p className="text-center">No items yet! Add one above!</p>
+    +                <p className="text-center">You have no todo items yet! Add one above!</p>
+    ```
+
+1. Let's build our updated version of the image, using the same command we used before.
+
+    ```bash
+    docker build -t getting-started .
+    ```
+
+1. Let's start a new container using the updated code.
+
+    ```bash
+    docker run -dp 3000:3000 getting-started
+    ```
+
+**Uh oh!** You probably saw an error like this (the IDs will be different):
+
+```bash
+docker: Error response from daemon: driver failed programming external connectivity on endpoint laughing_burnell 
+(bb242b2ca4d67eba76e79474fb36bb5125708ebdabd7f45c8eaf16caaabde9dd): Bind for 0.0.0.0:3000 failed: port is already allocated.
+```
+
+So, what happened? We aren't able to start the new container because our old container is still
+running. The reason this is a problem is because that container is using the host's port 3000 and
+only one process on the machine (containers included) can listen to a specific port. To fix this, 
+we need to remove the old container.
+
+
+## Replacing our Old Container
+
+To remove a container, it first needs to be stopped. Once it has stopped, it can be removed. We have two
+ways that we can remove the old container. Feel free to choose the path that you're most comfortable with.
+
+
+### Removing a container using the CLI
+
+1. Get the ID of the container by using the `docker ps` command.
+
+    ```bash
+    docker ps
+    ```
+
+1. Use the `docker stop` command to stop the container.
+
+    ```bash
+    # Swap out <the-container-id> with the ID from docker ps
+    docker stop <the-container-id>
+    ```
+
+1. Once the container has stopped, you can remove it by using the `docker rm` command.
+
+    ```bash
+    docker rm <the-container-id>
+    ```
+
+>**Pro tip**
```

Comment:
Could you update this to 'Note' instead?

Reply:
Yep updated from 'pro tip' to note.